sentry-android-core/src/test/java/io/sentry/android/core/DefaultAndroidEventProcessorTest.kt

```diff
@@ -165,14 +168,12 @@ class DefaultAndroidEventProcessorTest {
     }
 
     @Test
-    fun `Processor sets sdkInfo in the event`() {
+    fun `Processor sets sdkVersion in the event`() {
         val processor = DefaultAndroidEventProcessor(context, fixture.options, fixture.buildInfo, mock())
         val event = SentryEvent()
         processor.process(event, null)
-        assertNotNull(event.debugMeta.sdkInfo)
-        assertEquals("test", event.debugMeta.sdkInfo.sdkName)
-        assertEquals(1, event.debugMeta.sdkInfo.versionMajor)
-        assertEquals(2, event.debugMeta.sdkInfo.versionMinor)
-        assertEquals(3, event.debugMeta.sdkInfo.versionPatchlevel)
+        assertNotNull(event.sdk)
+        assertEquals("test", event.sdk.name)
+        assertEquals("1.2.3", event.sdk.version)
```

Comment:
`AndroidOptionsInitializerTest.init should set SdkVersion` already test if all the values are set.
`DefaultAndroidEventProcessorTest` only asserts if the object itself is copied, I've changed to `assertSame` instead of asserting its values.

Reply:
Okay didn't see the other test. My bad.